game/fonts/Multifont - Testsong/Multifont - Testsong.txt

```diff
@@ -0,0 +1,56 @@
+#TITLE:Testsong
```

Comment:
- [ ] please move this to the songs folder, or add it to https://github.com/UltraStar-Deluxe/songs